deploy/helm/charts/templates/lvm-controller.yaml

```diff
@@ -0,0 +1,146 @@
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - openebs-lvm-controller
+            topologyKey: "kubernetes.io/hostname"
+      priorityClassName: system-cluster-critical
+      serviceAccount: {{ .Values.serviceAccount.lvmController.name }}
+      containers:
+        - name: {{ .Values.lvmController.resizer.name }}
+          image: "{{ .Values.lvmController.resizer.image.registry }}{{ .Values.lvmController.resizer.image.repository }}:{{ .Values.lvmController.resizer.image.tag }}"
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: {{ .Values.lvmController.resizer.image.pullPolicy }}
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+          resources:
+            {{- toYaml .Values.lvmController.resources | nindent 12 }}
+        - name: {{ .Values.lvmController.snapshotter.name }}
+          image: "{{ .Values.lvmController.snapshotter.image.registry }}{{ .Values.lvmController.snapshotter.image.repository }}:{{ .Values.lvmController.snapshotter.image.tag }}"
+          imagePullPolicy: {{ .Values.lvmController.snapshotter.image.pullPolicy }}
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+          resources:
+            {{- toYaml .Values.lvmController.resources | nindent 12 }}
+        - name: {{ .Values.lvmController.snapshotController.name }}
+          image: "{{ .Values.lvmController.snapshotController.image.registry }}{{ .Values.lvmController.snapshotController.image.repository }}:{{ .Values.lvmController.snapshotController.image.tag }}"
+          args:
+            - "--v=5"
+            - "--leader-election=true"
+          imagePullPolicy: {{ .Values.lvmController.snapshotController.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.lvmController.resources | nindent 12 }}
+        - name: {{ .Values.lvmController.provisioner.name }}
+          image: "{{ .Values.lvmController.provisioner.image.registry }}{{ .Values.lvmController.provisioner.image.repository }}:{{ .Values.lvmController.provisioner.image.tag }}"
+          imagePullPolicy: {{ .Values.lvmController.provisioner.image.pullPolicy }}
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--v=5"
+            - "--feature-gates=Topology=true"
+            - "--strict-topology"
+            - "--leader-election"
+            - "--enable-capacity={{ .Values.storageCapacity }}"
```

Comment:
Is this required in latest kubernetes version also?

Reply:
Yes as per [this documentation](https://github.com/kubernetes-csi/external-provisioner#storage-capacity-arguments) (may be outdated). Anyways we can keep it configurable for sometime providing breathing space for latest upgrades.